### PR TITLE
New package: JiroShimizu.Jw_cad version 8.25.1.0

### PR DESCRIPTION
**Pull request by aurigajp**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/78721)

### CODE REVIEW

File: manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.locale.ja-JP.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS1.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: JiroShimizu.Jw_cad
+PackageVersion: 8.25a
+PackageLocale: ja-JP
+Publisher: Jiro Shimizu & Yoshifumi Tanaka
```
Comment (ItzLevvie) [inline manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.locale.ja-JP.yaml:7]:
```suggestion
Publisher: Jw_cad
```
This must match Control Panel.

File: manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.locale.ja-JP.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS1.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: JiroShimizu.Jw_cad
+PackageVersion: 8.25a
```
Comment (ItzLevvie) [inline manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.locale.ja-JP.yaml:5]:
```suggestion
PackageVersion: 8.25.1.0
```
This must match Control Panel.

File: manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS1.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: JiroShimizu.Jw_cad
+PackageVersion: 8.25a
```
Comment (ItzLevvie) [inline manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.installer.yaml:5]:
```suggestion
PackageVersion: 8.25.1.0
```
This must match Control Panel.

File: manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS1.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: JiroShimizu.Jw_cad
+PackageVersion: 8.25a
```
Comment (ItzLevvie) [inline manifests/j/JiroShimizu/Jw_cad/8.25a/JiroShimizu.Jw_cad.yaml:5]:
```suggestion
PackageVersion: 8.25.1.0
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by aurigajp — COMMENTED** [review]
fix the mistake
-> version
-> publisher

**Pull request closed without merging**